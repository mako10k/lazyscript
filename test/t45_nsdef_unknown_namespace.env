# Ensure this test runs without strict-effects
unset LAZYSCRIPT_ARGS
