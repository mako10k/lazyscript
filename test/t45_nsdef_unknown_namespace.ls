!{
  # nsdef on unknown named namespace should error
  (~~nsdef NonExist .x 1)
};
