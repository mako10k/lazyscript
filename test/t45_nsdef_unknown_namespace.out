E: prelude: unknown symbol: nsdef
E: <bottom msg="prelude: unknown symbol" at <unknown>:1.1: >
